app/src/lib/git/format-patch.ts

```diff
@@ -0,0 +1,26 @@
+import { git } from './core'
+import { revRange } from './rev-list'
+import { Repository } from '../../models/repository'
+
+/**
+ * Generate a patch containing the changes associated with this range of commits
```

Comment:
:art: tightening this wording up a bit, feel free to tweak or decline it

```suggestion
 * Generate a patch representing the changes associated with a range of commits
```